### DISCUSSION TITLE
1.10.0-alpha release

### DISCUSSION THREAD

**Discussion by SkalskiP** [SkalskiP/make-sense#279] [Announcements]
# [1.10.0-alpha] - 2022-09-30

## 🚀 Added

- Support for YOLOv5 models. (https://github.com/SkalskiP/make-sense/pull/276) You can now use pre-trained or custom models to speed up the annotation process. Take a look at the short video highlighting the new feature.

https://user-images.githubusercontent.com/26109316/193255987-2d01c549-48c3-41ae-87e9-e1b378968966.mov

## 🔧 Fixed

- Fix UI bugs that allowed to interact with hidden labels (https://github.com/SkalskiP/make-sense/pull/272)

## 🏆 Contributors

Special thanks are due to @IuliuNovac for his original [PR](https://github.com/SkalskiP/make-sense/pull/220) showcasing integration with [YOLOv5](https://github.com/ultralytics/yolov5). We most definitely would not have that killer feature in makesense.ai without your spark. 💥 Thank you! 🙏

## 🚧 Work in progress

One of the definite highlights of the next release will be the introduction of the long-awaited [stats view](https://github.com/SkalskiP/make-sense/issues/76), which would present statistics about annotated objects and images. The work on that feature is already quite advanced. (https://github.com/SkalskiP/make-sense/pull/268)

## What's Changed
* Add twitch link to Make Sense landing page by @SkalskiP in https://github.com/SkalskiP/make-sense/pull/270
* fix/it_is_possible_to_interact_with_hidden_labels by @SkalskiP in https://github.com/SkalskiP/make-sense/pull/272
* feature/156_import_custom_model_yolov5 by @SkalskiP in https://github.com/SkalskiP/make-sense/pull/276
* 1.10.0-alpha relese by @SkalskiP in https://github.com/SkalskiP/make-sense/pull/277
* 1.10.0-alpha relese by @SkalskiP in https://github.com/SkalskiP/make-sense/pull/278

**Full Changelog**: https://github.com/SkalskiP/make-sense/compare/1.9.0-alpha...1.10.0-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/SkalskiP/make-sense/releases/tag/1.10.0-alpha'>1.10.0-alpha release</a>.</em>

**Comment by IuliuNovac** [comment]
That looks amazing!

**Reply by SkalskiP** [reply]
Thank you for those kind words! But most importantly thank you for your 💡 idea!
